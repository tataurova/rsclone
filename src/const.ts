export const RouteName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};

export const AuthorizationStatus = {
  AUTH: 'AUTH',
  NO_AUTH: 'NO_AUTH',
};

export const Error = {
  UNAUTHORIZED: 401,
  BAD_REQUEST: 400,
  NOT_FOUND: 404,
  SERVICE_UNAVAILABLE: 503,
  TIMEOUT_ERROR: 'timeout_error',
};

export const headCells = [
  { id: 'status', label: 'Статус' },
  { id: 'city', label: 'Город' },
  { id: 'name', label: 'ФИО' },
  { id: 'age', label: 'Возраст' },
  { id: 'date', label: 'Дата' },
  { id: 'coordinator', label: 'Координатор' },
  { id: 'people', label: 'Людей на поиске' },
  { id: 'actions', label: 'Действия', disableSorting: true },
];

export const getStatusCollection = [
  'Резерв',
  'Поиск на месте',
  'Стоп. Проверка информации',
  'Архив',
];

export const SearcherStatus = {
  way: 'В пути',
  place: 'На месте',
  home: 'Дома',
};

export const TransportStatus = {
  car: 'На машине',
  own: 'Своим ходом',
  need: 'Нужна машина',
};

export const MAX_TIMEOUT = 5000;
export const SHOW_ERROR_TIMEOUT = 3000;
