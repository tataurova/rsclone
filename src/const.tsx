export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
