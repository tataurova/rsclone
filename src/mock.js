const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2017',
    sign: 'Был с собакиой',
    close: 'Шорты, белоя майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2018',
    sign: 'Хромает на леаую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2010',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
];

export default data;
