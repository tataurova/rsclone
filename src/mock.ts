export const data = [
  {
    id: 1,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 2,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Пертов Петр Петрович',
    city: 'Москва',
    birth: '04.09.1985',
    age: '30',
    missing: '01.10.2020',
    sign: 'Был с собакиой',
    close: 'Шорты, белая майка',
    with: 'черная сумка',
  },
  {
    id: 3,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Кузмин Картой Петрович',
    city: 'Судлино',
    birth: '30.12.1960',
    age: '60',
    missing: '05.12.2020',
    sign: 'Хромает на левую ногу',
    close: 'Шуба, валенки',
    with: 'трость',
  },
  {
    id: 4,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Щаруго Кирилл Федорович',
    city: 'Пирам',
    birth: '14.07.2000',
    age: '20',
    missing: '05.10.2020',
    sign: 'Заикается',
    close: 'Кепка синяя, белая майка',
    with: 'короткая стрижка',
  },
  {
    id: 5,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2020',
    age: '25',
    missing: '15.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 6,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '25.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 7,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    sign: 'Шрам на левой щеке',
    missing: '05.12.2020',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 8,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.04.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 9,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '05.02.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },
  {
    id: 10,
    image: 'http://lorempixel.com/280/220/people',
    name: 'Иванов Иван Иванович',
    city: 'Минск',
    birth: '02.05.2001',
    age: '25',
    missing: '14.05.2020',
    sign: 'Шрам на левой щеке',
    close: 'Куртка красная, джинсы синие',
    with: 'сумка спортивная',
  },

];

export const mockActiveSearches = [
  {
    id: 1,
    status: 'Поиск на месте',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чиж',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }, {
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  }, {
    id: 3,
    status: 'Стоп. Проверка информации',
    city: 'Минск',
    name: 'Аркадий',
    age: '100',
    date: '01-01-2021',
    coordinator: 'Чип',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 4,
    status: 'Поиск на месте',
    city: 'Москва',
    name: 'Владимир',
    age: '3000',
    date: '12-06-2020',
    coordinator: 'Дейл',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
  {
    id: 5,
    status: 'Резерв',
    city: 'Санкт-Петербург',
    name: 'Петр',
    age: '30',
    date: '12-01-2020',
    coordinator: 'Чи',
    people: [],
  },
  {
    id: 6,
    status: 'Поиск на месте',
    city: 'Киев',
    name: 'Анна',
    age: '123',
    date: '12-01-2020',
    coordinator: 'Кат',
    people: [{
      name: 'Саша',
      status: 'way',
      transport: 'car',
      places: '1',
      time: '12:00',
      district: 'Василеостровский',
    }, {
      name: 'Катя',
      status: 'place',
      transport: 'own',
      places: '0',
      time: '19:00',
      district: 'Василеостровский',
    }, {
      name: 'Рома',
      status: 'place',
      transport: 'need',
      places: '0',
      time: '2:00',
      district: 'Калининский',
    }, {
      name: 'Илья',
      status: 'way',
      transport: 'own',
      places: '0',
      time: '15:00',
      district: 'Выборгский',
    }],
  },
];
export const mockClosedSearches = [{
  id: 2,
  status: 'Архив',
  city: 'Колпино',
  name: 'Фекла',
  age: '200',
  date: '10-01-2020',
  coordinator: 'Чиж',
  people: [{
    name: 'Саша',
    status: 'way',
    transport: 'car',
    places: '1',
    time: '12:00',
    district: 'Василеостровский',
  }, {
    name: 'Катя',
    status: 'place',
    transport: 'own',
    places: '0',
    time: '19:00',
    district: 'Василеостровский',
  }, {
    name: 'Рома',
    status: 'place',
    transport: 'need',
    places: '0',
    time: '2:00',
    district: 'Калининский',
  }, {
    name: 'Илья',
    status: 'way',
    transport: 'own',
    places: '0',
    time: '15:00',
    district: 'Выборгский',
  }],
}];
