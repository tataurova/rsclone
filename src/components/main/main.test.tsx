import * as React from 'react';
import * as renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import Main from './main';
import { mockActiveSearches, mockClosedSearches } from '../../mock';
import { AuthorizationStatus } from '../../const';

jest.mock('../../i18n', () => ({
  addResourceBundle: jest.fn(),
}));

jest.mock('react-i18next', () => ({
  useTranslation: () => ({ t: (key) => key }),
}));

describe('<Main />', () => {
  const mockStore = configureStore([]);
  const store = mockStore({
    DATA: {
      gonePeople: [],
      lookingRelativesPeople: [],
      activeSearches: mockActiveSearches,
      closedSearches: mockClosedSearches,
      isFetching: false,
      error: false,
    },
    APP: {
      page: '2',
    },
    AUTH: {
      authorizationStatus: AuthorizationStatus.NO_AUTH,
      user: '',
    },
  });

  it('Render Main', () => {
    const tree = renderer
      .create(
          <Provider store={store}>
              <Main
                  page={'0'}
              />
          </Provider>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
