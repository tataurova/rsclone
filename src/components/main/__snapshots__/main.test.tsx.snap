// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Render Main 1`] = `
Array [
  <div
    className="makeStyles-root-6"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-7 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              onClick={[Function]}
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
        >
          Main
        </h6>
        <img
          alt="Logo"
          className="makeStyles-img-9"
          height="50px"
          src=""
          width="50px"
        />
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Login
          </span>
        </button>
      </div>
    </header>
  </div>,
  <main>
    <div
      className="MuiPaper-root makeStyles-mainSectionPost-3 MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "backgroundImage": "url(https://source.unsplash.com/random)",
        }
      }
    >
      <div
        className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-8"
          >
            <div
              className="makeStyles-mainSectionPostContent-4"
            >
              <h1
                className="MuiTypography-root MuiTypography-h3 MuiTypography-colorInherit MuiTypography-gutterBottom"
              >
                If you get lost in the woods
              </h1>
              <p
                className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit MuiTypography-paragraph"
              >
                If you get lost in the forest and do not see suitable landmarks, it is best to go out on the water and move downstream. The stream will definitely lead to the river, the river - to the people. If you get lost and know that you will be searched for, stay in one place and make a fire so the search engines will be easier to navigate.
              </p>
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label"
                >
                  <a
                    className="makeStyles-link-5"
                  >
                    Leave a request
                  </a>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiPaper-root makeStyles-mainSectionPost-3 MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "backgroundImage": "url(https://source.unsplash.com/random)",
        }
      }
    >
      <div
        className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-8"
          >
            <div
              className="makeStyles-mainSectionPostContent-4"
            >
              <h1
                className="MuiTypography-root MuiTypography-h3 MuiTypography-colorInherit MuiTypography-gutterBottom"
              >
                Rules of behavior of the child, if he is lost
              </h1>
              <p
                className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit MuiTypography-paragraph"
              >
                If the child realized that he was left alone, but his mother was just there, you need to call her loudly. The most important rule that you need to teach your baby do not agree to the suggestions of strangers to go look for parents. However, it is important that the child can ask for help from the seller, representatives of the security service or the police
              </p>
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label"
                >
                  <a
                    className="makeStyles-link-5"
                  >
                    Leave a request
                  </a>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiPaper-root makeStyles-mainSectionPost-3 MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "backgroundImage": "url(https://source.unsplash.com/random)",
        }
      }
    >
      <div
        className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-8"
          >
            <div
              className="makeStyles-mainSectionPostContent-4"
            >
              <h1
                className="MuiTypography-root MuiTypography-h3 MuiTypography-colorInherit MuiTypography-gutterBottom"
              >
                Rules that will help the elderly
              </h1>
              <p
                className="MuiTypography-root MuiTypography-h5 MuiTypography-colorInherit MuiTypography-paragraph"
              >
                If a person has problems with orientation and sense of space, they should always have identification documents or a special bracelet with their name and address engraved on it. Before leaving, a person should warn where he is going and when he will return approximately. You need to take your phone with you and put it in your inner pocket. When an elderly person has a phone with them, it is possible to keep in touch with them or track their location using a GPS signal.
              </p>
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label"
                >
                  <a
                    className="makeStyles-link-5"
                  >
                    Leave a request
                  </a>
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>,
  <footer
    className="makeStyles-footer-10"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
      >
        © 
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkFont-12 MuiTypography-colorInherit"
          href="https://rs.school/js/"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_blank"
        >
          RS School
        </a>
         
        <span
          className="makeStyles-flex-11"
        >
          <span
            className="makeStyles-link-13"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-14"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-16 MuiTypography-colorPrimary"
              href="https://github.com/tataurova/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Olga Tataurova
            </a>
          </span>
          <span
            className="makeStyles-link-13"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-14"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-16 MuiTypography-colorPrimary"
              href="https://github.com/shuliakleonid/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Leonid Shuliak
            </a>
          </span>
          <span
            className="makeStyles-link-13"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-14"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-16 MuiTypography-colorPrimary"
              href="https://github.com/andreizayaz"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Andrei Podyslenkau
            </a>
          </span>
        </span>
        2021
      </p>
    </div>
  </footer>,
]
`;
